src/components/callout/callout.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import Callout from './callout';
+import ShootingStar from '../../illustrations/shooting-star.svg';
+import ConstellationBigDipper from '../../illustrations/constellation-big-dipper.svg';
+
+const EXAMPLE_ILLUSTRATIONS = {
+  None: null,
+  ShootingStar: <ShootingStar />,
+  ConstellationBigDipper: <ConstellationBigDipper />,
+};
+
+export default {
+  component: Callout,
+  title: 'Components/Callout',
+  parameters: {
+    componentSubtitle:
+      'Display additional information detached from your surrounding text content',
+  },
+  argTypes: {
+    illustration: {
+      options: Object.keys(EXAMPLE_ILLUSTRATIONS),
+      mapping: EXAMPLE_ILLUSTRATIONS,
+      description:
+        'This is an example list of icons, there is not default set. You have to provide a svg react component/',
```

Comment:
Not sure about this `not` and this `/`...
```suggestion
        'This is an example list of icons, there is no default set. You have to provide a svg react component.',
```

Reply:
oh boy, I'm sorry for those stupid text errors. Next time I will check them more carefully. Thank you.